dotcom-rendering/src/web/components/ImageComponent.tsx

```diff
@@ -337,8 +337,10 @@ export const ImageComponent = ({
 					isLazy={!isMainMedia}
 					isMainMedia={isMainMedia}
 				/>
-				{starRating && <PositionStarRating rating={starRating} />}
-				{title && (
+				{typeof starRating === 'number' && (
+					<PositionStarRating rating={starRating} />
+				)}
```

Comment:
Here we want to handle a rating of `0`, as [it is valid and happens](http://content.guardianapis.com/search?api-key=test&page-size=100&show-fields=starRating&star-rating=0).